drivers/bridge/bridge.go

```diff
@@ -754,6 +755,14 @@ func parseEndpointOptions(epOptions map[string]interface{}) (*EndpointConfigurat
 		}
 	}
 
+	if opt, ok := epOptions[options.ExposedPorts]; ok {
```

Comment:
You did not take care of the test code. How can the test pass if it was feeding portmapping for link/unlink tests but now driver discards that ?